packages/typed-contracts/src/index.ts

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+import * as fs from 'fs';
+import {sync as globSync} from 'glob';
+import * as Handlebars from 'handlebars';
+import * as _ from 'lodash';
+import * as mkdirp from 'mkdirp';
+import * as yargs from 'yargs';
+
+import * as toSnakeCase from 'to-snake-case';
+import * as Web3 from 'web3';
+
+import {ParamKind} from './types';
+import {utils} from './utils';
+
+const args = yargs
+    .option('abiGlob', {
+        describe: 'Glob pattern to search for ABI JSON files',
+        type: 'string',
+        demand: true,
+    })
+    .option('templates', {
+        describe: 'Folder where to search for templates',
+        type: 'string',
+        demand: true,
+    })
+    .option('output', {
+        describe: 'Folder where to put the output files',
+        type: 'string',
+        demand: true,
+    })
+    .argv;
+
+function writeOutputFile(name: string, renderedTsCode: string): void {
+    const fileName = toSnakeCase(name);
+    const filePath = `${args.output}/${fileName}.ts`;
+    fs.writeFileSync(filePath, renderedTsCode);
+    utils.log(`Created: ${chalk.bold(filePath)}`);
+}
+
+Handlebars.registerHelper('parameterType', utils.solTypeToTsType.bind(utils, ParamKind.Input));
+Handlebars.registerHelper('returnType', utils.solTypeToTsType.bind(utils, ParamKind.Output));
+const partialTemplateFileNames = globSync(`${args.templates}/partials/**/*.mustache`);
+for (const partialTemplateFileName of partialTemplateFileNames) {
+    const namedContent = utils.getNamedContent(partialTemplateFileName);
+    Handlebars.registerPartial(namedContent.name, namedContent.content);
+}
+
+const mainTemplate = utils.getNamedContent(`${args.templates}/contract.mustache`);
```

Comment:
Move `contract.mustache` to constant at top of file. This should probably also be configurable perhaps? If not, then at least throw a descriptive error message if no file is found with that name. This will help the newbies!